RationaleMCP/0031/differences.md

```diff
@@ -1,6 +1,44 @@
 # Semantical differences between Flat Modelica and Modelica
 This document describes differences between Flat Modelica and Modelica that aren't clear from the differences in the grammars.
 
+## Lexical scoping inside record definitions
```

Comment:
Should contain two subsection covering
inside records
not treating records as packages

Reply:
Done.